packages/ckeditor5-build-helpjuice-v2/src/plugins/inlinecode/inlinecodeui.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+import { ButtonView } from 'ckeditor5/src/ui';
+import type AttributeCommand from '@ckeditor/ckeditor5-basic-styles/src/attributecommand';
+
+const INLINECODE = 'inlineCode';
+const INLINECODE_COMMAND = 'inlineCodeCommand';
```

Comment:
Minor, but can we reuse the definitions from `inlinecodeediting.ts`. This way, we need to change only one place in order to update the constant value.

Reply:
Done